torch/testing/_internal/common_cuda.py

```diff
@@ -121,3 +121,12 @@ def wrapped(self, device, dtype):
 
         return wrapped
     return wrapper
+
+
+def with_tf32_off(f):
```

Comment:
Comment explaining this function + example use here

Reply:
Is `with_tf32_off` really so interesting, can `tf32_off` not be used as a decorator?

```
@tf32_off
def test_foo...
```

Reply:
I don't understand. Do you mean to rename `with_tf32_off` to `tf32_off`? Currently, the `with_tf32_off` is for external usage, and `tf32_off` is for code readability.

Reply:
Just adding an explanatory comment + example is fine, but both @ngimel and I aren't sure we understand why `tf32_off` itself isn't used as a decorator directly.

Reply:
Comments added. `tf32_off` is a context manager, and it accepts no arguments. It is possible to modify `tf32_off` to make it a decorator and use it, but I don't think this way is clearer than the current approach.

Reply:
Sure. That's fine.